en/docs/learn/enterprise-integration-patterns/messaging-channels/datatype-channel.md

```diff
@@ -1,158 +1,382 @@
-| Receiver                                | Stock Quote Server Instance                                             |
-
-## ESB configuration
-
-Given below is the ESB configuration of this sample. Log in to the Management Console of the ESB profile, click Main, and then click Source View in the Service Bus menu to view this. 
-
-```
-<definitions xmlns="http://ws.apache.org/ns/synapse">
-   <endpoint name="StockQuoteReceiver">
-       <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
-   </endpoint>
-   <sequence name="MarketActivity">
-       <log level="custom">
-           <property name="Messaging_Channel" value="MARKET_ACTIVITY"/>
-       </log>
-       <send>
-           <endpoint key="StockQuoteReceiver"/>
-       </send>
-   </sequence>
-   <sequence name="FullQuote">
-       <log level="custom">
-           <property name="Messaging_Channel" value="FULL_QUOTE"/>
-       </log>
-       <send>
-           <endpoint key="StockQuoteReceiver"/>
-       </send>
-   </sequence>
- 
-   <sequence name="StockQuote">
-       <log level="custom">
-           <property name="Messaging_Channel" value="STOCK_QUOTE"/>
-       </log>
-       <send>
-           <endpoint key="StockQuoteReceiver"/>
-       </send>
-   </sequence>
-   <proxy name="datatype-channel-proxy" startOnLoad="true" transports="http https">
-       <target>
-           <log/>
-           <inSequence>
-               <switch source="get-property('Action')">
-                   <case regex="/*urn:getQuote/*">
-                       <sequence key="StockQuote"/>
-                   </case>
-                   <case regex="/*urn:getFullQuote/*">
-                       <sequence key="FullQuote"/>
-                   </case>
-                   <case regex="/*urn:getMarketActivity/*">
-                       <sequence key="MarketActivity"/>
-                   </case>
-               </switch>
-           </inSequence>
-           <outSequence>
-               <respond/>
-           </outSequence>
-       </target>
-   </proxy>
-</definitions>
-```
+| Sender                          | Client                                       |
+| Datatype Channel                | Switch and Sequence mediators specify which datatype channel to use. The Switch Mediator determines the appropriate datatype channel, and each datatype channel is defined as a Sequence Mediator.                                             |
+| Receiver                        | Stock Quote Server Instance                  |
+
+## Synapse configuration of the artifacts
+
+!!! note
+    When you unzip the ZIP file you downloaded below in step 7 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/wso2mi/artifacts` directory. For more information about these artifacts, go to WSO2 MI Documentation.
```

Comment:
'For more information about these artifacts, go to WSO2 MI Documentation.' Let's point to synapse artifacts page, as we are already in MI docs

Reply:
let's point to https://mi.docs.wso2.com/en/latest/develop/intro-integration-development/